syntax_and_semantics/literals/symbol.md

```diff
@@ -1,45 +1,62 @@
 # Symbol
 
-A [Symbol](http://crystal-lang.org/api/Symbol.html) is a constant that is identified by a name without you having to give it a numeric value.
+A [Symbol](http://crystal-lang.org/api/Symbol.html) represents a unique name inside the entire source code.
+It is internally implemented as a constant with a numeric value of type `Int32`.
```

Comment:
I think that going into implementation detail before any sample is not the best order.